src/lib/holocene/stories/Button.stories.ts

```diff
@@ -0,0 +1,42 @@
+import type { Meta, StoryObj } from '@storybook/svelte';
+
+import { icons } from '$lib/holocene/icon/paths';
+
+import ButtonWrapper from './ButtonWrapper.svelte';
+
+const iconControls = {
+  control: {
+    type: 'select',
+  },
+  options: Object.keys(icons),
+};
+
+// More on how to set up stories at: https://storybook.js.org/docs/svelte/writing-stories/introduction
+const meta = {
+  title: 'Example/Button',
+  component: ButtonWrapper,
+  argTypes: {
+    leadingIcon: { ...iconControls },
+    trailingIcon: { ...iconControls },
+    variant: {
+      control: {
+        type: 'select',
+      },
+      options: ['primary', 'secondary', 'destructive', 'ghost'],
+    },
```

Comment:
Maybe a future project, try to type this such that we can only have controls with valid prop names/values.

Reply:
100% They kind of were before but not sure why this wasn't typed correctly. I think it ends up being a weird generic thing